spec.md

```diff
@@ -385,21 +385,12 @@ It is an error if the value of an octal or hexadecimal escape is greater than de
 "\x41-\x5A"             # "A-Z"
 ```
 
-The bit width of each string element is not defined by this spec,
-and for reasons of efficiency and interoperability with the host language,
-it varies across implementations.
-For example, in the Go and Rust implementations,
-each string element is an 8-bit value (a byte) and Unicode text is encoded as UTF-8,
-whereas in the Java implementation,
-string elements are 16-bit values (Java `char`s) and Unicode text is encoded as UTF-16.
-This spec generalizes both approaches by referring to the string encoding as "UTF-K",
-where the value of K is implementation-defined.
```

Comment:
We can't push the definition of "UTF-K" below its first use.

Reply:
Good point: I forgot that the literals are presented before the data type they represent.

Tweaked.